src/client/api/pool.c

```diff
@@ -38,6 +38,12 @@ daos_pool_connect(const char *pool, const char *sys, unsigned int flags,
 
 	return dc_task_schedule(task, true);
 }
+#undef daos_pool_connect
+int
+daos_pool_connect(const char *pool, const char *sys, unsigned int flags,
+                  daos_handle_t *poh, daos_pool_info_t *info, daos_event_t *ev)
+__attribute__ ((weak,alias ("daos_pool_connect2")) );
```

Comment:
(style)  space prohibited between function name and open parenthesis '('
(style)  space required after that ',' (ctx:VxV)
(style)  space prohibited before that close parenthesis ')'